### PR TITLE
Updating fields for new share buttons.

### PR DESCRIPTION
**Pull request by ScottDowne**
Related issue:
Summary:

If you don't know who to ask for a review, try @alanmoo or @xmatthewx. If you do, please delete this line.

### CODE REVIEW

File: network-api/networkapi/campaign/models.py [hunk 1]
```diff
@@ -85,13 +85,33 @@ class Petition(models.Model):
...
     share_link_text = models.CharField(
         max_length=20,
         help_text="Text content of the share button",
         default="Share this",
+        blank=True,
+        editable=False
+    )
+
+    share_twitter = models.CharField(
+        max_length=20,
```
Comment (gideonthomas) [inline network-api/networkapi/campaign/models.py]: check limit

File: network-api/networkapi/campaign/models.py [hunk 2]
```diff
@@ -85,13 +85,33 @@ class Petition(models.Model):
...
+        help_text="Share Progress id for twitter button",
+        blank=True
+    )
+
+    share_facebook = models.CharField(
+        max_length=20,
```
Comment (gideonthomas) [inline network-api/networkapi/campaign/models.py]: check limit

File: network-api/networkapi/campaign/models.py [hunk 3]
```diff
@@ -85,13 +85,33 @@ class Petition(models.Model):
...
+        help_text="Share Progress id for facebook button",
+        blank=True
+    )
+
+    share_email = models.CharField(
+        max_length=20,
```
Comment (gideonthomas) [inline network-api/networkapi/campaign/models.py]: check limit

File: network-api/networkapi/campaign/models.py (hunk 1 above)
Comment (ScottDowne) [reply network-api/networkapi/campaign/models.py]: Limit is 9, and it roughly looks like this "sp_185606" I'm happy to leave it at 20 tbh, but if you want me to tighten it up to 9 I can do that too.

**Pull request merged**